.dockerignore

```diff
@@ -0,0 +1,31 @@
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file for more about ignoring files.
```

Comment:
- [ ] Kindly, resolve the review comment left on the frontend app.

Reply:
Thanks @Meltrust I will make the changes 🙏